03week/towersOfHanoi.js

```diff
@@ -13,33 +13,54 @@ let stacks = {
   c: []
 };
 
-function printStacks() {
+let startStack = ['a', 'b', 'c'];
+let endStack = ['a','b','c'];
+
+const printStacks = ()=> {
   console.log("a: " + stacks.a);
   console.log("b: " + stacks.b);
   console.log("c: " + stacks.c);
 }
-
-function movePiece() {
-  // Your code here
-
-}
-
-function isLegal() {
-  // Your code here
-
+//Fucntion to move disks
+const movePiece = (startStack,endStack)=> {
+  return stacks[endStack].push(stacks[startStack].pop());
 }
-
-function checkForWin() {
-  // Your code here
-
+//Function to check if move is legal
+const isLegal = (startStack, endStack)=> {
+  let startTest = stacks[startStack][stacks[startStack].length - 1];
+  let endTest = stacks[endStack][stacks[endStack].length - 1];
+
+  if ((startTest < endTest) || (stacks[endStack].length === 0)) {
```

Comment:
this can be simplified to `return (startTest < endTest) || (stacks[endStack].length === 0)`